.github/workflows/helm-chart-ci.yaml

```diff
@@ -173,6 +173,7 @@ jobs:
               --namespace "${scenario}" \
               --target-branch ${{ github.base_ref }} \
               --exclude-deprecated \
+              --skip-clean-up \
```

Comment:
The summary's are failing or inconsistent because the results are getting cleaned up before the summary code runs.

Reply:
This has been fixed with a new release.